lib/services/presets.go

```diff
@@ -203,6 +203,7 @@ func defaultAllowRules() map[string][]types.Rule {
 			types.NewRule(types.KindSAMLIdPServiceProvider, RW()),
 			types.NewRule(types.KindOktaImportRule, RW()),
 			types.NewRule(types.KindOktaAssignment, RW()),
+			types.NewRule(types.KindDevice, RO()),
```

Comment:
i double checked with marco, that whatever rule we defined in our preset template, it should be the same exact rule when including it here (so whether you are starting a brand new cluster or have existing cluster and upgrading, they both will have the same rules): https://github.com/gravitational/teleport/blob/6490eb88c09d9d2720c6f18ed74fb9f450006bcb/lib/services/presets.go#L78

Reply:
I thought to make the `RO()` only since the UI only has list functionality for now. But noted your point. Updating that now.

Reply:
Can we add a comment in the code to make sure this doesn't happen again in the future?

Reply:
I added comment as well.